### DISCUSSION TITLE
Pre-built optimized docker image

### DISCUSSION THREAD

**Discussion by NicoGuilhem** [keycloak/keycloak#32567] [Ideas]
Hello,

With Quarkus and dev services, default image used is the official from `quay.io/keycloak/keycloak` wich is not "optimized" as suggested in [documentation](https://www.keycloak.org/server/containers).

Would it be possible to ship an official image already "optimized" for this kind of usage ?

See related discussion on quarkusio : https://github.com/quarkusio/quarkus/issues/42906

Thanks :)

**Comment by shawkins** [comment]
@NicoGuilhem there isn't a one size fits all optimized image. Provide 1 or more would probably lead to a proliforation of images - users would at least want one image per supported database type. Adding a consideration for any other build time option would just further multiple the number we'd be expected to provide.

**Comment by NicoGuilhem** [comment]
@shawkins thanks for your answer, i understand.
